src/main/java/io/jenkins/plugins/extlogging/api/Event.java

```diff
@@ -15,6 +15,7 @@
     final long timestamp;
     final long id;
 
+    //TODO:Consider <String,String> so that Key-Value storages could be quickly implemented
```

Comment:
Yes please; I do not think implementations can be expected to take arbitrary Java objects.